Cargo.toml

```diff
@@ -1,3 +1,3 @@
 [workspace]
 
-members = ["xline", "curp", "benchmark", "utils", "engine", "xlineapi"]
+members = ["xline", "curp", "benchmark", "utils", "engine", "xlineapi", "curp-extapi", "curp-testutils"]
```

Comment:
I think curp-external-api would be more readable.

Reply:
done